projects/ngx-mapbox-gl/src/lib/source/image-source.component.ts

```diff
@@ -17,42 +17,38 @@ export class ImageSourceComponent implements OnInit, OnDestroy, OnChanges, Image
   @Input() url: string;
   @Input() coordinates: number[][];
 
-  private sourceAdded = false;
-  private sub = new Subscription();
+  private readonly dead$ = new Subject();
```

Comment:
While this pattern is fine (I do use this pattern in large apps too :D), it's not used anywhere in this lib (as far as I recall). For consistency, I believe it's better to stick to the classic manual unsubscribe, especially when there's only one subscription.

Reply:
Gotcha. Will fix it.

Reply:
Same thing for the example, for consistency.

Reply:
Well I do use it in draggable directive, but in my defense, there's a lot a subscriptions there :sweat_smile:

Reply:
That's fine. I'm all hands for consistency 👍 . Btw, it might be a good idea to have prettier (or some other formatter) setup for the project, so it's easier to be consistent (+ things like "formatOnSave" & "organizeImports" in .vscode/settings) . What do you think?

Reply:
Yep, prettier totally planned for v4 :)